corehq/apps/userreports/reports/specs.py

```diff
@@ -484,6 +485,32 @@ def get_fields(self, data_source_config=None, lang=None):
         return self.numerator.get_fields() + self.denominator.get_fields()
 
 
+class ArrayAggLastValueReportColumn(ReportColumn):
```

Comment:
I was not sure if `LastValueColumn` would confuse the reader that this is using the last_value function like other classes do, hence included `ArrayAgg` in the name.

Reply:
> using the last_value function like other classes do

What do you mean?  The only references I see to `last_value` in `corehq/apps/userreports` relate to this class.

Reply:
Sorry, I had missed responding to this.

My intention behind that name was to make it explicit that it was using a combination of array agg and last value as the implementation.

> Also, the fact that this uses an array aggregation is an implementation detail that's largely irrelevant to how the class is used externally

That sounds reasonable though I'd like to keep that special case of using array agg explicit.

More context:
Postgres also has a [last_value function](https://www.postgresqltutorial.com/postgresql-last_value-function/), and I think sqlagg/in reports we do naming such that its the `<SQL_Function>Column` which is what i was trying to avoid conflict with. If we were to also add support simply for that function along with this implementation, that is the case this explicit distinction would come in handy.